packages/api/cms-api/src/translation/translation.module.ts

```diff
@@ -0,0 +1,26 @@
+import { DynamicModule, Global, Module } from "@nestjs/common";
+
+import { TranslationServiceInterface } from "./translation.config";
+import { TranslationResolver } from "./translation.resolver";
+import { TRANSLATION_CONFIG } from "./translation-config.constants";
+
+interface TranslationConfig {
+    service: TranslationServiceInterface;
+}
+
+@Global()
+@Module({})
+export class TranslationModule {
```

Comment:
Please rename to `ContentTranslationModule` to be in line with the naming in the Admin

Reply:
Resolved outdated.